src/main/java/io/confluent/connect/elasticsearch/RetryUtil.java

```diff
@@ -170,6 +170,10 @@ protected static <T> T callWithRetries(
 
         log.warn("Failed to {} due to {}. Retrying attempt ({}/{}) after backoff of {} ms",
             description, e.getCause(), attempt, maxTotalAttempts, backoff);
+        log.trace("Failed to {} due to {} and stacktrace {}. "
+                         + "Retrying attempt ({}/{}) after backoff of {} ms",
+                         description, e.getCause(), e.getStackTrace(), attempt,
+                         maxTotalAttempts, backoff);
```

Comment:
```suggestion
        log.trace("Retrying {} attempt ({}/{}) after backoff of {} ms. Failed due to {}",
                         description, attempt, maxTotalAttempts, backoff, e);
```
Im not sure if `e.getStackTrace()` will print the traces of of causes. Best to just log e.

Reply:
Ok, agree we can get more info too.